# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/systemservices/objc_video_capture/device_info_objc.mm
# Commit: 703360f0b861
# Full Hash: 703360f0b8611916dfb3724eda7f35345d3f4be7
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2022-12-13 04:11:09
# Regressor Bug: 1451394
# File Overlap Count: 1
# Description:
#   Bug 1451394 - Integrate with the libwebrtc camera backend for Mac. r=webrtc-reviewers,jib
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D163684
# ==============================================================================

diff -r 49002523b70d -r 703360f0b861 dom/media/systemservices/objc_video_capture/device_info_objc.mm
--- a/dom/media/systemservices/objc_video_capture/device_info_objc.mm	Mon Dec 12 15:47:22 2022 +0000
+++ b/dom/media/systemservices/objc_video_capture/device_info_objc.mm	Mon Dec 12 15:47:22 2022 +0000
@@ -15,7 +15,6 @@
 #import <AVFoundation/AVFoundation.h>
 
 #import "device_info_objc.h"
-#include "modules/video_capture/video_capture_config.h"
 
 @implementation DeviceInfoIosObjC
 
@@ -30,7 +29,7 @@
 - (void)dealloc {
 }
 
-- (void)registerOwner:(DeviceInfoIos*)owner {
+- (void)registerOwner:(webrtc::VideoCaptureModule::DeviceInfo*)owner {
   [_lock lock];
   if (!_owner && owner) {
     [self configureObservers];